haptools/index.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+import logging
+from pathlib import Path
+from haptools.data.haplotypes import Haplotypes
+from pysam import tabix_index
+from haptools import data
+import tempfile
+
+"""
```

Comment:
this docstring looks great, but I think it should go between [line 28-29](https://github.com/gymrek-lab/haptools/pull/89/files#diff-b1e9e9accd453c676ce526b74039e77754a95f3d7292c4bc52b274c2f05f658bR28) underneath the function definition